### PR TITLE
CodeFormatter Run

### PR DESCRIPTION
**Pull request by jaredpar**
Fell out of our normal cadence for this during the push for RTM.  Now that we
had a bit more breathing room getting us back on track here.

### CODE REVIEW

**Comment by jaredpar** [comment]
CC @agocke, @dpoeschl, @VSadov

File: src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs [hunk 1]
```diff
@@ -3812,7 +3812,7 @@ public void CscUtf8Output_WithRedirecting_On()
 
             var output = RunAndGetOutput("cmd", "/C \"" + s_CSharpCompilerExecutable + "\" /utf8output /nologo /preferreduilang:en /t:library " + srcFile + " > " + tempOut.Path, expectedRetCode: 1);
             Assert.Equal("", output.Trim());
-            Assert.Equal("SRC.CS(1,1): error CS1056: Unexpected character '♚'", tempOut.ReadAllText().Trim().Replace(srcFile, "SRC.CS"));
+            Assert.Equal("SRC.CS(1,1): error CS1056: Unexpected character '\u265A'", tempOut.ReadAllText().Trim().Replace(srcFile, "SRC.CS"));
```
Comment (CyrusNajmabadi) [inline src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs]: This seems unfortunate.  What's the reason to change these characters into escapes?

File: src/Diagnostics/FxCop/Core/Design/CA1052DiagnosticAnalyzer.cs [hunk 2]
```diff
@@ -44,20 +44,20 @@ public class CA1052DiagnosticAnalyzer : AbstractNamedTypeAnalyzer
     {
         public const string DiagnosticId = "CA1052";
 
-        private static readonly LocalizableString Title = new LocalizableResourceString(
+        private static readonly LocalizableString s_title = new LocalizableResourceString(
```
Comment (CyrusNajmabadi) [inline src/Diagnostics/FxCop/Core/Design/CA1052DiagnosticAnalyzer.cs]: Does your system detect conflicts?

File: src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs (hunk 1 above)
Comment (jaredpar) [inline src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs]: @CyrusNajmabadi the default behavior in codeformatter is to escape unicode characters. I neglected to turn the behavior off when I did this run.

File: src/Diagnostics/FxCop/Core/Design/CA1052DiagnosticAnalyzer.cs (hunk 2 above)
Comment (jaredpar) [inline src/Diagnostics/FxCop/Core/Design/CA1052DiagnosticAnalyzer.cs]: @CyrusNajmabadi it uses the Roslyn rename API directly

Comment (CyrusNajmabadi) [inline src/Diagnostics/FxCop/Core/Design/CA1052DiagnosticAnalyzer.cs]: /giphy nooice

**Pull request merged**